### PR TITLE
Add slovenia debarment crawler

### LINKED ISSUE #419: Add Slovenia debarments
Debarments from public procurement in Slovenia: https://erar.si/omejitve/
Already crawled by OCCRP Aleph: https://aleph.occrp.org/datasets/8867

### PR DESCRIPTION
**Pull request by atehe**
Closes #419  

@jbothma I crawled the dataset similar to the entities used in [aleph](https://aleph.occrp.org/datasets/8867) but the `target.nested.json` stays empty.
I was thinking it is could be because the debarment role has not been added to the dataset but the target entity (`LegalEntity`)  does not have th attribute.
can you please take a look and let me know what you think.

### CODE REVIEW

**Comment by jbothma** [comment]
hey! I get 11048 entities, 5368 targets. Could it be that when you're exporting it's not re-generating the "store"? Sorry - I have a docs improvement that we'll merge soon that talks about that https://github.com/opensanctions/opensanctions/pull/495/files#diff-be685f78c6a9454eb7878fa369e723f2aae3430e9b013a54ef8a85edf915e57bR35

**Comment by atehe** [comment]
Ah... thanks, it exports now.
Where can I add the role ,`debarment` to the dataset .
I was thinking the `UnknownLink` entity will suffice?

File: datasets/si/br/crawler.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    if not record.get("do").startswith("9999"):
+        validity_link.add("endDate", record.get("do"))
+
+    validity_link.add(
+        "description",
+        "When the restriction is valid from, and when it ends. No end date implies it is valid until canceled",
+    )
+    validity_link.add("subject", legal_entity)
+    validity_link.add("object", organization)
```
Comment (jbothma) [inline datasets/si/br/crawler.py]: totally see why you added the org and the link here.
in this case let's only do the legal entity, and the debarment topic, and note why these entities are debarred in the entity program property.

**Review by jbothma — CHANGES_REQUESTED** [review]
Just a quick look at this. I'll take a better look tomorrow.

File: datasets/si/br/si_br.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  in which an official, or their family member, holds a managerial, administrative, or legal role, or has a significant financial stake exceeding five percent of founding rights, management, 
+  or capital. This measure is designed to prevent conflicts of interest and ensure transparency in public dealings.
+
+  This list is compiled using data provided by committed authorities, with officials required to report information about related entities within one month of assuming office and within 
+  eight days of any subsequent changes. Once received, the relevant authority must communicate this data to the Commission within 15 days. Consequently, the process of entering a specific 
+  entity onto the list may take up to 45 days from the start of a function, and changes may take up to 23 days. 
+
+  It's crucial to note that business restrictions come into effect by law immediately upon their occurrence, irrespective of whether the entity is officially registered on the list. 
+  Therefore, the absence of an entity on the list does not negate the applicability of business restrictions with that entity.
```
[the remark this answers is not in the document]
Comment (jbothma) [reply datasets/si/br/si_br.yml:19]: Could you see if you can summarise from https://www.kpk-rs.si/en/restrictions-of-business-activities/ replace the More detail from the publisher section, and then also link to that page similarly to how I have in that section?

**Review by jbothma — CHANGES_REQUESTED** [review]
alright - our journalist contact has also pointed out that this is quite a complex one, but that we understand correctly. Nice job figuring it out.

I've proposed a much briefer form as the opening section to the description, and our contact provided a link to an english explanation page so we don't have to use the translated detail - we can paraphrase from the clearer website and link to it.

thanks again!

**Comment by atehe** [comment]
The description you added is verry clear.
I've updated it with the publishers details and data source.

File: datasets/si/br/si_br.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+title: Slovenia Business Restrictions
+entry_point: crawler.py
+prefix: si-br
+url: https://erar.si/omejitve/
+summary: List of Business restrictions preventing conflict of interest in Slovenia
+description: >
+  Slovenian public bodies and organizations may not procure from entities in which an official in that organization or their family member is a manager, legal representative, or has 5% or more ownership.
+  This is a list of entities where such a relationship exists, and the public organization is restricted/prohibited or permitted to engage in business activities in some instances with the entity. The restricted public organization is noted in the program property of each entity. **The restriction is only on the public organization where the public official has a role - it does not apply to the entire public sector.**
+  **This is a preventative restriction by Komisija za preprečevanje korupcije and implies no wrongdoing.**
```
Comment (jbothma) [inline datasets/si/br/si_br.yml:9]:
```suggestion
  Slovenian public bodies and organizations may not procure from entities in which an official in that organization or their family member is a manager, legal representative, or has 5% or more ownership.
  
  This is a list of entities where such a relationship exists, and the public organization is restricted/prohibited or permitted to engage in business activities in some instances with the entity. The restricted public organization is noted in the program property of each entity. **The restriction is only on the public organization where the public official has a role - it does not apply to the entire public sector.**
  
  **This is a preventative restriction by Komisija za preprečevanje korupcije and implies no wrongdoing.**
```

File: datasets/si/br/si_br.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  More detail from the publisher:
+  ----------------------------------
+  As the public sector conducts business with private-sector entities, certain corruption risks are generated, especially when individuals with personal interests in the business activity are involved. To safeguard public funds from being used for private gain, legal measures, such as restrictions on business activities and prohibitions, are implemented.
+  These measures are outlined in the Integrity and Prevention of Corruption Act (IPCA) and focus on preventing corruption and avoiding conflicts of interest.
+  [Translated and edited from https://www.kpk-rs.si/en/restrictions-of-business-activities/](https://www.kpk-rs.si/en/restrictions-of-business-activities/)
```
Comment (jbothma) [inline datasets/si/br/si_br.yml:15]:
```suggestion
  ### More detail from the publisher:
  
  > As the public sector conducts business with private-sector entities, certain corruption risks are generated, especially when individuals with personal interests in the business activity are involved. To safeguard public funds from being used for private gain, legal measures, such as restrictions on business activities and prohibitions, are implemented.
  > These measures are outlined in the Integrity and Prevention of Corruption Act (IPCA) and focus on preventing corruption and avoiding conflicts of interest.
  [From https://www.kpk-rs.si/en/restrictions-of-business-activities/](https://www.kpk-rs.si/en/restrictions-of-business-activities/)
```

File: datasets/si/br/si_br.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  Data gathering process
+  --------------------------
+  The data is gathered from the app [Erar](https://erar.si/doc/). It is a web application developed by the Komisija za preprečevanje korupcije to enhance transparency by allowing the public and state authorities in Slovenia to access detailed information on transactions involving public institutions and state-owned entities.
+  While the Commission emphasizes the accuracy of published data, occasional discrepancies may occur, urging users to check their completeness, accuracy and up-to-dateness with the original administrator of the database before making decisions based on them. Due to such corrections, the published data may change at any time without prior notice.
```
Comment (jbothma) [inline datasets/si/br/si_br.yml:20]:
```suggestion
  ### Data gathering process

    The data is gathered from the app [Erar](https://erar.si/doc/). It is a web application developed by the Komisija za preprečevanje korupcije to enhance transparency by allowing the public and state authorities in Slovenia to access detailed information on transactions involving public institutions and state-owned entities.
  While the Commission emphasizes the accuracy of published data, occasional discrepancies may occur, urging users to check their completeness, accuracy and up-to-dateness with the original administrator of the database before making decisions based on them. Due to such corrections, the published data may change at any time without prior notice.
```

**Review by jbothma — COMMENTED** [review]
Thank you!

Next time please add to the appropriate collection too.

**Comment by atehe** [comment]
Noted JD.
Can you please confirm the appropraite collection you are referring to

**Comment by jbothma** [comment]
In this case it was debarments, which was easy, because there's a debarments collection. It's not always obvious, but a lot of the time there's a direct match. https://zavod.opensanctions.org/tutorial/#add-your-crawler-to-a-collection

**Comment by atehe** [comment]
Understood now, thanks

**Pull request merged**